include/RE/M/MenuTopicManager.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include "RE/B/BGSSceneAction.h"
+#include "RE/B/BSFixedString.h"
+#include "RE/B/BSString.h"
+#include "RE/B/BSTEvent.h"
+#include "RE/B/BSTList.h"
+#include "RE/B/BSTSingleton.h"
+
+namespace RE
+{
+	class BGSDialogueBranch;
+	class DialoguePackage;
+	class MenuEventHandler;
+	class MenuOpenCloseEvent;
+	class TESQuest;
+	class TESTopic;
+	class TESTopicInfo;
+	struct PositionPlayerEvent;
+
+
+	class MenuTopicManager :
+		public BSTSingletonSDM<MenuTopicManager>,	// 0x10
+		public BSTEventSink<MenuOpenCloseEvent>,	// 0x00
+		public BSTEventSink<PositionPlayerEvent>	// 0x08
+	{
+	public:
+		inline static constexpr auto RTTI = RTTI_MenuTopicManager;
+
+
+		struct Dialogue
+		{
+		public:
+			struct ResponseData
+			{
+			public:
+				// members
+				BSString					responseText;	 // 00
+				BGSSceneAction::EmotionType emotionType;	 // 10
+				std::uint32_t				emotionValue;	 // 14
+				BSFixedString				voiceFilePath;	 // 18
+				std::uint64_t				unk20;			 // 20
+				std::uint64_t				unk28;			 // 28
+				std::uint64_t				unk30;			 // 30
+				bool						useEmotionAnim;	 // 38
+				bool						hasLipFile;		 // 39
+				std::uint16_t				pad3A;			 // 3A
+				std::uint32_t				pad3C;			 // 3C
+			};
+			static_assert(sizeof(ResponseData) == 0x40);
```

Comment:
This is a duplicate of [this](https://github.com/Ryan-rsm-McKenzie/CommonLibSSE/blob/master/include/RE/D/DialogueItem.h#L21-L51).

Reply:
Oh, didn't saw that one.